pkg/pool-hooks/contracts/MevHook.sol

```diff
@@ -39,6 +47,9 @@ contract MevHook is BaseHooks, SingletonAuthentication, VaultGuard, IMevHook {
     // Global max dynamic swap fee percentage returned by this hook.
     uint256 internal _maxMevSwapFeePercentage;
 
+    // Global list of senders that bypass MEV tax and pay static fees.
+    EnumerableSet.AddressSet internal _mevTaxExemptSenders;
```

Comment:
I think I'd just use a mapping here; should be enough.
The most important thing about this is to query whether an address is in the mapping or not; we don't care about the total allowlisted addresses onchain.